### ISSUE TITLE
Allow users to enter spaces or commas when inputting values

### ISSUE THREAD

**Issue by chrisdwheatley** [chrisdwheatley/wowsuch#32]
Currently any spaces or commas in the input boxes will stop any calculations happening and in turn stop any values displaying.

Need to work around this by possibly removing commas and spaces (and possibly any other characters which might be used) on the fly. An Angular filter sounds like the right way to go but need to look into possible options a bit more.

**Comment by jimthedev** [comment]
Good idea. Some thoughts in the comments of this reply in terms of restricting keys: http://stackoverflow.com/a/2824914/1397590

**Comment by chrisdwheatley** [comment]
Thanks for that link, it looks like it'll help achieve exactly what's required.
